src/components/topic/TopicDirective.js

```diff
@@ -31,19 +31,25 @@ goog.require('ga_topic_service');
               }
             });
 
-            gaTopic.loadConfig().then(function() {
-              scope.topics = gaTopic.getTopics();
-              scope.activeTopic = gaTopic.get();
-              scope.$applyAsync(function() {
-                element.find('.ga-topic-item').tooltip({
-                  container: modal,
-                  placement: 'bottom'
+            $rootScope.$on('$translateChangeEnd', function() {
+              scope.topics = gaTopic.sort(gaTopic.getTopics());
+            });
+
+            $translate.onReady(function() {
+              gaTopic.loadConfig().then(function() {
```

Comment:
I'm  against  the use of  $translate.onReady because that means we need to wait for translations to load topics and we don't have to wait, that slow down the loading of the page. And if the translations are broken the topic will not load.

I suggest to remove the $translate.onReady and use the translate in the HTML partials as before.